activestorage/test/models/attachment_test.rb

```diff
@@ -134,6 +134,15 @@ class ActiveStorage::AttachmentTest < ActiveSupport::TestCase
     assert_equal blob, ActiveStorage::Blob.find_signed(signed_id)
   end
 
+  test "can destroy attachment without existing relation" do
+    blob = create_blob
+    @user.highlights.attach(blob)
+    attachment = @user.highlights.find_by(blob_id: blob.id)
+    attachment.update_attribute(:name, 'old_highlights')
+    assert_equal attachment.send(:dependent), nil
+    attachment.destroy
```

Comment:
Presumably it would be better to setup an expectation like:
`assert_nothing_raised { attachment.destroy }`

Reply:
That's definitely more explicit to the intentions of that line. Thanks.